src/common/components/game/GameNotification.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { bool, string } from 'prop-types';
+import Notification from 'react-web-notification';
+
+const GameNotification = ({ enabled, text }) => {
+  const options = {
+      tag: 'wordbots',
+      icon: '/static/icons/android-icon-144x144.png',
```

Comment:
Should this image be imported dynamically like:
```
import wordBotsIcon from '/static/icons/android-icon-144x144.png';
```

Reply:
Actually I think `react-web-notification` does that under the hood.